include/deal.II/matrix_free/operators.h

```diff
@@ -636,6 +636,44 @@ namespace MatrixFreeOperators
           const VectorizedArray<Number> *               in_array,
           VectorizedArray<Number> *                     out_array) const;
 
+    /**
+     * This operation performs a projection from the data given in quadrature
+     * points to the actual basis underlying this object. This projection can
+     * also be interpreted as a change of the basis from the Lagrange
+     * interpolation polynomials in the quadrature points and to the
+     * representation by the basis underlying the current `fe_eval` object.
```

Comment:
I am not sure that I understand this sentence correctly. Do you want to say
```
[...] as a change of the basis from the Lagrange
interpolation polynomials in the quadrature points to the
representation in the basis underlying the current `fe_eval` object.
```
?

Reply:
Yes, but I will write it as 
```
as a change of the basis from the Lagrange
interpolation polynomials in the quadrature points to the
basis underlying the current `fe_eval` object.
```